### PR TITLE
Prefs [Task][27962]: Handle bind data for Detail Countries Information screen

### PR DESCRIPTION
**Pull request by NguyenTaTrung**
## Related Tickets
- [#Task_27962: Handle bind data for Detail Countries Information screen](https://edu-redmine.sun-asterisk.vn/issues/27962)

## WHAT
- Create view, presenter, adapter
- Bind data: get data pour into textview

## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Trainer reivew (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the source code have end-line errors? | | NO |<li>- [x] no</li>|<li>- [ ] no</li>  
Conventions | No hard coding, use constants/configuration values | | YES |<li>- [x] yes</li>|<li>- [ ] yes</li>  
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [x] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [x] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [x] yes</li>|<li>- [ ] yes</li>

### CODE REVIEW

File: app/src/main/java/com/sunasterisk/anticovid_19/ui/detail/DetailCountriesFragment.kt [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    override fun showError(error: String) {
+        context?.showToast(error)
+    }
+
+    override fun showLoading() = myDialog.show()
+
+    override fun hideLoading() = myDialog.hide()
+
+    private fun initAdapter() {
+        recyclerViewCountries.layoutManager = LinearLayoutManager(context)
```
Comment (huytq-2422) [inline app/src/main/java/com/sunasterisk/anticovid_19/ui/detail/DetailCountriesFragment.kt]: cái này set trong xml được ko

Comment (NguyenTaTrung) [reply app/src/main/java/com/sunasterisk/anticovid_19/ui/detail/DetailCountriesFragment.kt]: Giời t mới biết là set đc :D

File: app/src/main/java/com/sunasterisk/anticovid_19/ui/detail/DetailCountriesFragment.kt [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+import com.sunasterisk.anticovid_19.ui.dialog.LoadingDialog
+import com.sunasterisk.anticovid_19.utils.RepositoryUtil
+import com.sunasterisk.anticovid_19.utils.showToast
+import kotlinx.android.synthetic.main.fragment_detail_countries.*
+
+class DetailCountriesFragment : BaseFragment(), DetailCountriesContract.View {
+
+    private var presenter: DetailCountriesPresenter? = null
+    private val adapter = DetailCountriesAdapter()
+    private lateinit var myDialog: LoadingDialog
```
Comment (quockhanhng) [inline app/src/main/java/com/sunasterisk/anticovid_19/ui/detail/DetailCountriesFragment.kt]: https://viblo.asia/p/co-nen-su-dung-kotlin-lateinit-var-hay-khong-LzD5dbneZjY

File: app/src/main/java/com/sunasterisk/anticovid_19/ui/statistic/StatisticsFragment.kt [hunk 3]
```diff
@@ -141,6 +130,13 @@ class StatisticsFragment : BaseFragment(),
...
 
+    private fun showFragment() {
+        (activity as MainActivity).supportFragmentManager
+            .beginTransaction()
+            .replace(R.id.fragmentContainer, DetailCountriesFragment.getInstance())
+            .commit()
+    }
+
     companion object {
         fun getInstance() = StatisticsFragment()
```
Comment (quockhanhng) [inline app/src/main/java/com/sunasterisk/anticovid_19/ui/statistic/StatisticsFragment.kt:145]: sao không tạo luôn bằng constructor luôn anh

Comment (NguyenTaTrung) [reply app/src/main/java/com/sunasterisk/anticovid_19/ui/statistic/StatisticsFragment.kt:145]: À a viết trc thôi về sau còn truyền dữ liệu nữa

**Pull request merged**